zebROS_ws/src/tf_object_detection/CMakeLists.txt

```diff
@@ -168,8 +168,9 @@ target_link_libraries(tf_object_screen_to_world
 ## Mark executable scripts (Python etc.) for installation
 ## in contrast to setup.py, you can choose the destination
  install(PROGRAMS
-   src/test_detection.py
+   src/trtdetection.py
    src/pub_video.py
+   src/trtdebug.py
```

Comment:
Probably can remove this if everything has been consolidated into the trtdetection script.